app/models/feedback.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Feedback
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  RATINGS = {
+    5 => 'Very satisfied',
+    4 => 'Satisfied',
+    3 => 'Neither satisfied nor dissatisfied',
+    2 => 'Dissatisfied',
+    1 => 'Very dissatisfied'
+  }.freeze
+
+  attr_accessor :comment, :rating, :email
+  validates :rating, inclusion: { in: RATINGS.keys.map(&:to_s), message: :invalid }
```

Comment:
I think you can just change the yml file error to be inclusion and you don't need this. did you try
```suggestion
  validates :rating, inclusion: { in: RATINGS.keys.map(&:to_s) }
```